man/systemd-repart.xml

```diff
@@ -619,6 +619,16 @@
         <xi:include href="version-info.xml" xpointer="v255"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--merge-fstab=<replaceable>BOOL</replaceable></option></term>
+
+        <listitem><para>Enables the merge of the current (if present)
```

Comment:
to me "merge" sounds like we'd do some kind of by-line analysis or so. But this here is just "dumb" appending, hence maybe clal it "--append-fstab="?

Reply:
Renamed.